ophyd/areadetector/filestore_mixins.py

```diff
@@ -169,6 +168,43 @@ def stage(self):
         self._resource = fs.insert_resource('AD_TIFF', self._fp, res_kwargs)
 
 
+class FileStoreTIFFSquashing(FileStoreBase):
+    def __init__(self, *args, images_per_set_name='images_per_set',
+                 number_of_sets_name="number_of_sets",
+                 cam_name='cam', proc_name='proc1', **kwargs):
+        super().__init__(*args, **kwargs)
+        self._ips_name = images_per_set_name
+        self._num_sets_name = number_of_sets_name
+        self._cam_name = cam_name
+        self._proc_name = proc_name
+
+    def stage(self):
+        # 'Single' means one image : one file. It does NOT mean that
+        # 'num_images' is ignored.
+        cam = getattr(self.parent, self._cam_name)
+        proc = getattr(self.parent, self._proc_name)
+        images_per_set = getattr(self.parent, self._ips_name).get()
+        num_sets = getattr(self.parent, self._num_sets_name).get()
+        self.stage_sigs.update([(self.file_template, '%s%s_%6.6d.tiff'),
+                                (self.file_write_mode, 'Single'),
+                                (proc.nd_array_port, cam.port_name.get()),
+                                (proc.reset_filter, 1),
+                                (proc.enable, 1),
```

Comment:
This is not necessary. The processing plugin enables itself during its own staging process.